bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -728,7 +728,7 @@ impl Connection {
     //
     /// Returns a reference to the ClientHello associated with the connection.
     /// ```compile_fail
-    /// use s2n_tls::client_hello::{ClientHello, FingerprintType};
+    /// use s2n_tls::client_hello::{fingerprint::FingerprintType, ClientHello};
```

Comment:
Won't this change break anyone currently using the fingerprinting feature? I think you need to "use" to make the fingerprinting methods still look like they're just in the client_hello module.

Maybe test how you'd use fingerprinting from another crate, like s2n-tls-tokio?

Reply:
Or a great excuse to add an example #4469  😄

Reply:
Fixed so that the path is unchanged from the perspective of a user.